bot.py

```diff
@@ -1,545 +1,748 @@
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['remcost'])
+async def remcost_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        query = message.text.replace('/remcost ', '').strip()
+        if changer(query, 0, 'cost.data') == True:
+            await message.reply(lng.getmess(message, "removed"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['addcost'])
+async def addcost_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        query = message.text.replace('/addcost ', '').strip()
+        if changer(query, 1, 'cost.data') == True:
+            await message.reply(lng.getmess(message, "added"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['listcode'])
+async def listcode_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        await message.reply(cost_code_reactor(message.text.replace('/listcode', '').strip(), 'code.data', full = True), parse_mode = "Markdown")
+
+@dp.message(commands = ['listcost'])
+async def listcode_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        await message.reply(cost_code_reactor(message.text.replace('/listcost', '').strip(), 'cost.data', full = True), parse_mode = "Markdown")
+
+@dp.message(commands = ['s', 'sr'])
+async def search_mess(message: types.Message):
+    if users.getstatus(message) == Status.AUTH or users.getstatus(message) == Status.MAYDAY:
+        com = '/sr' if '/sr' in message.text else '/s'
+        query = message.text.replace(com, '').replace(f'@{BOT_USERNAME}', '').strip()
+        if query == '':
+            await message.reply(lng.getmess(message, "empty"), parse_mode = "Markdown")
+        else:
+            try:
+                macs = [m.upper().replace('-', ':') for m in re.findall(MAC, query)]
+                for m in macs:
+                    query = query.replace(m, '').strip()
+            except:
+                query = query.strip()
+            finally:
+                    p = {'key': users.data[str(message.from_user.id)]['key'] if users.getstatus(message) == Status.AUTH else ADMIN_KEY, 
+                        'essid': query if query != [] else '*', 
+                        'bssid': [m for m in macs] if macs != [] else '*',
+                        'sens': True if com == '/sr' else False
+                        }
+                    r = requests.post('https://3wifi.stascorp.com/api/apiquery', json = p).json()
```

Comment:
```suggestion
                    r = requests.post(f'{SERVICE_URL}/api/apiquery', json = p).json()
```
Сорри, переносил код, не изменил.